app/controllers/application_controller.rb

```diff
@@ -10,11 +10,7 @@ class ApplicationController < ActionController::API
 
   def with_current_request
     ManageIQ::API::Common::Request.with_request(request) do |current|
-      begin
-        ActsAsTenant.with_tenant(current_tenant(current.user)) { yield }
-      rescue ManageIQ::API::Common::IdentityError
-        json_response({ :message => 'Unauthorized' }, :unauthorized)
-      end
+      ActsAsTenant.with_tenant(current_tenant(current.user)) { yield }
```

Comment:
I like this change, and for context on line 5 of the `application_controller` is where the `rescue_from` is located.
`rescue_from ManageIQ::API::Common::IdentityError, :with => :unauthorized_error`

My question: The removed code was responded with a message of 'Unauthorized' which is safer in my mind than
```
 def unauthorized_error(err)
    render :json => {:message => err.message}, :status => :unauthorized
  end
```
`err.message` that the unauthorized_error returns.  I wonder if we should just change the message for `unauthorized_error` to 'Unauthorized' and log the `err.message`.  Thoughts?

Reply:
Probably the same could be true for the `forbidden_error` as well.

Reply:
That sounds good to me. That way we still can read the long error but the user just sees a generic message.

Reply:
I did this change in #263 since I moved them over there.